bot/exts/moderation/modpings.py

```diff
@@ -3,21 +3,22 @@
 
 import arrow
 from async_rediscache import RedisCache
-from dateutil.parser import isoparse, parse as dateutil_parse
+from dateutil.parser import isoparse
 from discord import Embed, Member
 from discord.ext.commands import Cog, Context, group, has_any_role
 
 from bot.bot import Bot
 from bot.constants import Colours, Emojis, Guild, Icons, MODERATION_ROLES, Roles
-from bot.converters import Expiry
+from bot.converters import DayDuration, Expiry
 from bot.log import get_logger
 from bot.utils import scheduling
+from bot.utils.members import get_or_fetch_member
 from bot.utils.scheduling import Scheduler
 from bot.utils.time import TimestampFormats, discord_timestamp
 
 log = get_logger(__name__)
 
-MAXIMUM_WORK_LIMIT = 16
+MAXIMUM_WORK_OFF_LIMIT = 16
```

Comment:
I am confused by this, why the requirement? I can't find where the discussion was held about - it's not linked anywhere on this PR.

Reply:
For reference, this was answered on discord in #dev-contrib [here](https://discord.com/channels/267624335836053506/635950537262759947/929923352901058601).